core/src/test/java/fr/sncf/osrd/envelope_sim_infra/MRSPTest.java

```diff
@@ -30,18 +32,18 @@
 public class MRSPTest {
     @Test
     public void speedLimitByCategory() {
-        final var edge = new TrackSectionImpl(100, "edge");
+        final var edge = new TrackSectionImpl(1000, "edge");
```

Comment:
The total length in the first test is already equal to the accelerating postponement offset.
This is why we multiplied the scale of the track by 10.
another solution would be to consider the the acceleration postponment offset to be 10 instead of 100 and the braking anticipation offset to be 5 instead of 50 when testing. what do you think @axrolld ?

Reply:
In the original message I wrongly assumed that the default values were (0, 0).

When modifying existing tests it *should* be set to (0, 0). We really don't want to change what happens in unrelated tests. Non-zero values should only be used in new tests.

Reply:
Well, it depends how we want to test the driver behavior. There are two options : 
- either we apply the driver behavior to every existing test (what has been done), but that implies making every trip longer otherwise a 100m braking anticipation on a 100m long path does not make sense anymore
- or we leave all the existing tests without the driver behavior, and we write some specific tests